crates/bevy_render/src/camera/projection.rs

```diff
@@ -161,49 +169,48 @@ pub enum ScalingMode {
     FixedHorizontal(f32),
 }
 
-/// Project a 3D space onto a 2D surface using parallel lines, i.e., objects have the same
-/// apparent size regardless of depth. The volume contained in the projection is called the view frustum.
+/// Project a 3D space onto a 2D surface using parallel lines, i.e., unlike [`PerspectiveProjection`],
+/// the size at which objects appear remain the same regardless of depth.
 ///
-/// If the window is smaller than the view frustum, some objects within the view frustum will not appear on the screen.
```

Comment:
Is there any way for the width/height of the view frustum to be different than the width/height of the window? The only case I can think of is if `scaling_mode` is `None`, but then contents in the view frustum will just be stretched to fill the window.

Reply:
@xgbwei Will the contents just be stretched to fit the window? I thought some things will just lie outside the window if the view frustrum is larger than the window, for example (and thus those objects will not be visible)?

Reply:
> Is there any way for the width/height of the view frustum to be different than the width/height of the window? The only case I can think of is if `scaling_mode` is `None`, but then contents in the view frustum will just be stretched to fill the window.

The `viewport` field on the camera allows you to specify custom viewport dimensions, for things like multiple views. In my tiled camera crate I use it to force the game view to a multiple of some target resolution to avoid pixel artifacts when rendering low resolution images.